pkg/osquery/runtime/runner.go

```diff
@@ -656,3 +660,62 @@ func newRunner(opts ...OsqueryInstanceOption) *Runner {
 		opts:     opts,
 	}
 }
+
+func (r *Runner) runOsqueryVersionCheckAddToKnapsack(ctx context.Context, osquerydPath string) {
+	var output bytes.Buffer
+
+	osq, err := runsimple.NewOsqueryProcess(osquerydPath, runsimple.WithStdout(&output))
+	if err != nil {
+		r.slogger.Log(ctx, slog.LevelError,
+			"unable to create process",
+			"err", err,
+		)
+		return
+	}
+
+	versionCtx, versionCancel := context.WithTimeout(ctx, 30*time.Second)
+	defer versionCancel()
+
+	startTime := time.Now().UnixMilli()
+
+	osqErr := osq.RunVersion(versionCtx)
+	executionTimeMs := time.Now().UnixMilli() - startTime
```

Comment:
I think you can remove `startTime` and `executionTimeMs` -- we cared about those in `launcher.go => runOsqueryVersionCheck` when we were trying to troubleshoot a particular bug, but we don't really care about logging the execution time here 🙂

Reply:
removed the execution time from runner